java/src/main/native/ai_onnxruntime_OrtSession_SessionOptions.c

```diff
@@ -464,6 +465,24 @@ JNIEXPORT void JNICALL Java_ai_onnxruntime_OrtSession_00024SessionOptions_addNup
   #endif
 }
 
+/*
+ * Class::    ai_onnxruntime_OrtSession_SessionOptions
+ * Method:    addStvm
+ * Signature: (JILjava/lang/String)V
+ */
+JNIEXPORT void JNICALL Java_ai_onnxruntime_OrtSession_00024SessionOptions_addStvm
```

Comment:
This doesn't do anything without the appropriate `private void native addStvm(long,long,String)` method in `OrtSession.SessionOptions`, as it won't appear in the autogenerated header file.

Reply:
@vvchernov, please address in a follow-up. thanks.

Reply:
@Craigacp @jywu-msft  Thanks! I will do it at the next stage of development.